lib/marqeta/user.rb

```diff
@@ -20,6 +20,11 @@ def children
       ApiObject.object_list(User, "users/#{token}/children")
     end
 
+    def onetime
+      result = ApiCaller.new('users/auth/onetime').post(user_token: token)
+      ApiObject.new(result)
```

Comment:
Hmm I'm not so sure about this. `ApiObject` was designed to be an abstract superclass. I think instead we should stick with the pattern that already exists and make a new subclass of `ApiObject` called `OneTime` that defines `self.endpoint` to return `'users/auth/onetime'`.

And then from here we'd call `OneTime.api_create(user_token: token)`

Reply:
I'm happy to discuss if you disagree, but I'm pretty sure that's the direction we should go.

Reply:
@dubroe ah! just like `perform_kyc`. I was following the `Card.from_pan` trick for some reason